foreman/data_refinery_foreman/surveyor/test_sra_xml.py

```diff
@@ -0,0 +1,388 @@
+EXPERIMENT_XML = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is huge. Do I need to check it line by line?

Reply:
You don't need to check it at all, except as far as it is useful for understanding the XML parsing.